src/ralph/lib/transitions/decorators.py

```diff
@@ -22,6 +25,13 @@ def decorator(func):
         @wraps(func)
         def wrapper(*args, **kwargs):
             return func(*args, **kwargs)
+
+        if 'model' in kwargs:
+            setattr(kwargs['model'], func.__name__, wrapper)
+            action, _ = Action.objects.get_or_create(name=func.__name__)
```

Comment:
this shouldn't be necessary if transition is registered before post_migrate signal